resources/interactive.mjs

```diff
@@ -229,7 +229,7 @@ function startTest()
     if (searchParams.has('suite'))
         disableAllSuitesExcept(searchParams.get('suite'));
 
-    const iterationCount = searchParams.get('iterationCount') || 1;
+    const iterationCount = Number(searchParams.get('iterationCount')) || 1;
```

Comment:
How about just +.

Reply:
Doesn't matter to me either way (I think this code will eventually be removed/merged into the main runner)